model/todo.py

```diff
@@ -0,0 +1,49 @@
+from .db import todos
+
+class TodoModel:
+
+    def get_todos(self):
+        return todos
+
+    def get_single_todo(self, todo_id):
+        single_todo = {}
+        for todo in todos:
+            if todo['id'] == todo_id:
+                single_todo = todo
+                return single_todo
+        return single_todo
+    
+    def create_todo(self, todo_object):
+        todos.insert(0, todo_object)
+        return todo_object
+
+
+    def delete_todo(self, todo_id):
+        todo_id = int(todo_id, base=10)
+        i = 0
```

Comment:
Awesome job @adex001 You don't have to use `i` to loop through. `for` will loop through the object

Reply:
alright. Thanks for your feedback. But is there any other way to get the index so I can use the pop method?

Reply:
Try `todos .remove(todo)`

Reply:
ok! I will.. Thanks!